package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
```

Comment:
Since we aren't really using node anymore, consider using [phony targets](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html) to do everything we require yarn to do. Then we'll really be node-free